test/StaticValidatorTest.php

```diff
@@ -154,7 +154,6 @@ public function testPassingNullWhenSettingPluginManagerResetsPluginManager()
 
     public function testExecuteValidWithParameters()
     {
-        $this->assertTrue(StaticValidator::execute(5, 'Between', [1, 10]));
```

Comment:
The second test here is actually better.

Originally it was testing for 5 being between 1 and 10, which would have always been true with the defaults.  It never tested for the negative case though.